crates/paimon/src/spec/types.rs

```diff
@@ -200,12 +239,24 @@ impl ArrayType {
     }
 }
 
+impl Serialize for ArrayType {
+    fn serialize<S>(
+        &self,
+        serializer: S,
+    ) -> Result<<S as serde::Serializer>::Ok, <S as serde::Serializer>::Error>
+    where
+        S: serde::Serializer,
+    {
+        serializer.serialize_str(&self.to_string())
```

Comment:
Can we use `#[serde_as(as = "DisplayFromStr")]` instead of implement `Serialize` by hand?